docs/lib/getting-started/fragments/android/setup.md

```diff
@@ -1,78 +1,110 @@
+    ![](~/images/lib/getting-started/android/set-up-android-studio-welcome.png)
 
-a. Open your **project** `build.gradle` and add the following:
-* `mavenCentral()` as a repository
-* `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'` as a dependency
-* A plugin `'com.amplifyframework.amplifytools'` as shown in the example below:
+1. In **Select a Project Template**, select **Empty Activity**. Press **Next**.
 
-```gradle
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
-        classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'
+    ![](~/images/lib/getting-started/android/set-up-android-studio-select-project-template.png)
+
+1. Next, configure your project:
+
+    - Enter *Todo* in the **Name** field
+    - Select either *Java* or *Kotlin* from the **Language** dropdown menu
+    - Select *API 16: Android 4.1 (Jelly Bean)* from the **Minumum SDK** dropdown menu
+    - Press **Finish**
+
+  ![](~/images/lib/getting-started/android/set-up-android-studio-configure-your-project.png)
+
+When finished, Android Studio will open your project with a tab opened to either *MainActivity.java* or *MainActivity.kt* depending upon if you created a Java or Kotlin project respectively.
+
+![](~/images/lib/getting-started/android/set-up-android-studio-successful-setup.png)
+
+### Add Amplify to your application
+
+Amplify for Android is distribued as an Apache Maven package. In this section, you'll add the packages and other required directives to your build configuration.
+
+1. Expand **Gradle Scripts** and open **build.gradle (Project: Todo)**. Add the following lines:
+
+    - Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'` within the `dependencies` block
+    - Add the line  `plugin 'com.amplifyframework.amplifytools'` at the end of the file 
+
+    ```groovy
+    buildscript {
+        ...
+
+        dependencies {
+            classpath 'com.android.tools.build:gradle:3.6.3'
+            classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'
+        }
     }
-}
 
-apply plugin: 'com.amplifyframework.amplifytools'
-```
+    apply plugin: 'com.amplifyframework.amplifytools'
+    ```
 
-b. Next, add the following dependencies to your **app** `build.gradle`. Note the declaration of `compileOptions`, to make use of Java 8 features like Lambda Expressions.
+    This configuration adds helpers to your IDE to allow easy generation and deployment of Amplify files and resources.
 
-```gradle
-android {
-    compileOptions {
-        sourceCompatibility 1.8
-        targetCompatibility 1.8
+1. Under **Gradle Scripts**, open **build.gradle (Module: app)**. Add the following lines:
+
+    - `sourceCompatibility` and `targetCompatibility` to Java 1.8 which allows your application to make use of Java 8 features like Lambda expressions
```

Comment:
Are they able to target higher than Java 1.8? If so... should we mention that they can target higher language versions?

Reply:
No, I think we basically want them to just copy and paste these exactly lines. @TrekSoft @jamesonwilliams please weigh in if I'm wrong.